src/App.jsx

```diff
@@ -68,10 +68,9 @@ function App() {
                     exact
                     path="/account"
                     element={
-                      <AccountPage
-                        changesMade={accMadeChanges}
-                        setChangesMade={setAccMadeChanges}
-                      />
+                      <ProtectedRoute redirectPath="/logout" roles={[ADMIN_ROLE, VOLUNTEER_ROLE]}>
+                        <AccountPage setChangesMade={setAccMadeChanges} />
+                      </ProtectedRoute>
```

Comment:
Instead of modifying the protected route, you can pass in the element into the Component prop as an arrow function. Please do this instead and revert ProtectedRoute back to how it was before.

`<ProtectedRoute Component={() => <AccountPage setChangesMade={...} />} ... >`